### ISSUE TITLE
"Tween is still active" error messages aren't user friendly 

### ISSUE THREAD

**Issue by balloonpopper** [godot-escoria/escoria-issues#134]
**Describe the bug**
An error with a tween still running at the end of the event results in a error message that isn't intuitive
e.g. Tween is still active: 3.982040/12.558296

**To Reproduce**
1. Do a very slow camera push
e.g. 
camera_set_pos 100 1500 150
wait 1
stop

**Expected behavior**
The error message was a little more helpful
e.g. 
```
The tween was still running when the event completed.
Tween time elapsed when event ended : [val]
Tween total time required : [val]
```

**Screenshots**
![image](https://user-images.githubusercontent.com/5151242/155267526-1f62e128-c5d2-475e-9960-51d1c57323ba.png)

**Versions**
* Escoria: latest
* Godot:  3.4.2

**Comment by dploeger** [comment]
I agree, though I'd set it to future, because it really is a bit cosmetic, if that's okay with you.

**Comment by balloonpopper** [comment]
It looks to be a small fix so I'll probably tackle it myself, but otherwise future is fine.
